Start-Up.md

```diff
@@ -0,0 +1,27 @@
+# Start-Up Procedure
```

Comment:
Maybe start at `##` header level, in case we ever want to merge topics together (see my review summary comment)